### ISSUE TITLE
Productize `sigma.js/node-rapids` `extra-large-graphs` to `sigma.js` quality and satisfaction.

### ISSUE THREAD

**Issue by thomcom** [jacomyal/sigma.js#1258]
## Complete PR and all review items.

I've produced https://www.github.com/jacomyal/sigma.js/pull/1252. Today I'm starting to revise it until there's nothing extra or unnecessary. The original PR was based on an experimental research branch that produced the fruit I was looking for. Now I want that PR to meet all of your expectations. I'm going to keep a checklist here for now, tracking my progress as I update the PR.

- [x]  Get rid of the `.glsl` shaders in the demo.

I moved these shaders from the `sigma.js` hierarchy because I was having trouble importing them. I'll spend time making that work as expected.

- [x]  Remove `generate-graph.ts`

Just gonna go with .js on this one.

- [x]  Add arguments and messaging to `generate-graph.js`

- [x]  Modify index.html to display graph parameters instead of take them.

- [x]  Consider a button to switch between GPU and CPU data ETL with benchmark.

The above is not going to happen in this PR, it'll take another PR if we get there. I'll definitely be looking at more benchmarks if there's interest.

- [x]  Clean up index.ts

- [x]  Document the hacks in `rapids-graph.ts` and the engine improvements they imply.

- [x]  Modify `package.json` to handle launching on a different port specifically for `extra-large-graphs`

- [x] Reset and minimize changes outside of `extra-large-graphs`.

- [x] Make a sweet README.md about what is going on here.

Depends on #1252

**Comment by thomcom** [comment]
It's done. Please take a look at https://www.github.com/jacomyal/sigma.js/pull/1252 and let me know what you think!

**Comment by Yomguithereal** [comment]
Thanks @thomcom, see my remarks on the PR. I will close this for now since the result is on the PR.
